config/locales/client.en.yml

```diff
@@ -34,6 +35,13 @@ en:
       channel_close:
         title: "Close Channel"
         instructions: "Closing the channel prevents non-staff users from sending new messages or editing existing messages. Are you sure you want to close this channel?"
+      channel_delete:
+        title: "Delete Channel"
+        instructions: "<p>Deletes the <strong>%{name}</strong> channel and chat history. All messages and related data, such as reactions and uploads, will be permanently deleted. If you want to preserve the channel history and decomission it, you may want to Archive the channel instead.</p>
```

Comment:
```suggestion
        instructions: "<p>Deletes the <strong>%{name}</strong> channel and chat history. All messages and related data, such as reactions and uploads, will be permanently deleted. If you want to preserve the channel history and decomission it, you may want to archive the channel instead.</p>
```

Reply:
I also wonder if it'll be helpful if we can link `archive` to the archiving modal? Only if it is easy :)